crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -171,3 +205,47 @@ pub struct Capsule {
 }
 impl super::Primitive2d for Capsule {}
 impl Primitive3d for Capsule {}
+
+impl Capsule {
+    /// Create a new `Capsule` from a radius and length
+    pub fn new(radius: f32, length: f32) -> Self {
+        Self {
+            radius,
+            half_length: length / 2.0,
+        }
+    }
+}
+
+/// A cone primitive.
+#[derive(Clone, Copy, Debug)]
+pub struct Cone {
+    /// radius of the base
+    pub radius: f32,
+    /// height of the cone
+    pub height: f32,
```

Comment:
These should be capitalized and have an article for consistency
```suggestion
    /// The radius of the base
    pub radius: f32,
    /// The height of the cone
    pub height: f32,
```